documentation/book/con-mutual-tls-authentication.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+// assembly-configuring-kafka-listeners.adoc
+
+[id='con-mutual-tls-authentication-{context}']
+= Mutual TLS authentication for clients
+
+== Mutual TLS authentication
+
+{ProductName} can configure Kafka to use TLS (Transport Layer Security) to provide encrypted communication between Kafka brokers and clients either with or without _mutual authentication_. 
+(Mutual TLS authentication is always used for the communication between Kafka brokers and zookeeper pods).
+
+Mutual authentication is when both parties to a communication get proof of the identity of the other party: The broker authenticates the client and the client authenticates the broker.
+
+NOTE: In many common uses of TLS (such as the HTTPS protocol used between a web browser and a web server) the authentication is not mutual: Only one party to the communication gets proof of the identity of the other party.
```

Comment:
I'm not sure I'd use "use case" like that. How about: 

> TLS authentication is more commonly one-way, where only one party authenticates to another. For example, when the HTTPS protocol is used between a web browser and a web server, the authentication is not usually mutual and only the server  gets proof of the identity of the browser.

(should this still be a NOTE? It's really an aside, since many people are only familiar with TLS from experience with HTTPS I think it's valuable to call out this difference, but it's tangential to the main thrust of the concept).

Reply:
@tombentley where else would you put it? Or are you saying remove it completely? 

I'm okay with removing if we don't need it.  If you change it to your suggestion, that's good too.